src/Controllers/SignOutVisitorByAdminController.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Controllers;
+
+use App\Abstracts\Controller;
+use App\Models\VisitorModel;
+use Psr\Http\Message\ResponseInterface as Response;
+use Psr\Http\Message\ServerRequestInterface as Request;
+
+class SignOutVisitorByAdminController extends Controller
+{
+    private VisitorModel $model;
+
+    public function __construct($model)
+    {
+        $this->model = $model;
+    }
+
+    public function __invoke(Request $request, Response $response, array $args)
+    {
+        $name = $request->getParsedBody();
+        $this->model->signOutVisitor($name);
+        return $response->withHeader('Location', '/admin');
```

Comment:
surely this controller and `SignOutVisitorController.php` are the same except the redirect? so you could do an if statement to see if they are logged in as an admin and decide which to redirect to? then ud only need 1 controller